src/graphapi/find_engine.cpp

```diff
@@ -81,7 +81,7 @@ class MHA_Fwd_F8_Pattern : public GraphPatternMatcher
     {
 
         assert(attn_scale);
-        std::vector<int64_t> all1s = {1LL, 1LL, 1LL, 1LL};
+        std::vector<std::size_t> all1s = {1LL, 1LL, 1LL, 1LL};
```

Comment:
`ull` for `size_t` literals.

Reply:
@amberhassaan should I do the following without C++ 23 uz?

`std::vector<std::size_t> all1s = {std::size_t{1}, std::size_t{1}, std::size_t{1}, std::size_t{1}};`

or we fine with 1UL?